lib/OpenQA/Utils.pm

```diff
@@ -348,43 +348,53 @@ my %bugrefs = (
     bsc => 'https://bugzilla.suse.com/show_bug.cgi?id=',
     boo => 'https://bugzilla.opensuse.org/show_bug.cgi?id=',
     poo => 'https://progress.opensuse.org/issues/',
+    gh  => 'https://github.com/',
 );
 my %bugurls = (
     'https://bugzilla.novell.com/show_bug.cgi?id=' => 'bsc',
     $bugrefs{bsc}                                  => 'bsc',
     $bugrefs{boo}                                  => 'boo',
     $bugrefs{poo}                                  => 'poo',
+    'https://github.com/'                          => 'gh',
```

Comment:
probably it differed during my development but does not anymore now. I will correct.